src/neuroglancer/ui/tool.ts

```diff
@@ -397,6 +408,25 @@ export class LayerToolBinder {
     this.clear();
   }
 
+  loadDefaultKeybinds() {
+    if (typeof DEFAULT_KEYBINDS !== 'undefined') {
+      const obj: DefaultKeybindsForLayerType = {};
+      const defaultKeybindsForLayer = DEFAULT_KEYBINDS[this.layer.type] || {};
+      for (const [key, value] of Object.entries(defaultKeybindsForLayer)) {
+        const toolAlreadyBound = [...this.bindings.values()].includes(value);
+        if (toolAlreadyBound) {
+          continue;
+        }
+        const existingTool = this.globalBinder.get(key);
+        if (existingTool && !existingTool.layer.managedLayer.archived) {
```

Comment:
I can see that it might potentially be desired, but is there a specific reason why you want to override an existing binding for an archived layer?

Reply:
I will remove that, I was unfamiliar with how to fully remove layers when I implemented that.